### DISCUSSION TITLE
[Enhancement Request] Permission/group based home worldlimits

### DISCUSSION THREAD

**Discussion by kristouffe** [Zrips/CMI#8310] [Enhancement]
Hi,
after long hours of fighting and discussion (**thanks Cloude_Lecaw**) I finally got the wordlimit feature for homes to work (I guess the expected way).

ATM, worldlimits in Homes.yml are settable for ALL PLAYERS at the same time.

But imagine we want specific players only (**groups**) to be able to set homes or not to set in a specific world.

**Example:**

- Group A has a total of 1 home, this home should be settable only in the _default world_. 
- Group B has a total of 2 homes, 1 should be allowed in the _default world_, and 1 in the _world_nether_.
- Group C has total of 5 homes, 2 should  be allowed in the _default world_, 2 in the _world_nether_, 1 in the _world_the_end_.

ATM this configuration is not possible.

**Adding permissions** per worldlimit group in Homes.yml could bring this feature to a whole new level and make the whole stuff much more customizable.

**Comment by CloudeLecaw** [comment]
It would be nice to see this.
This would be a good Improvement of the way how we can customize sethomes for the worlds
